storageClient/api.go

```diff
@@ -34,6 +34,7 @@ type Note struct {
 type CreateGroupRequest struct {
 	Name              string `json:"group_name"`
 	PublicKey         string `json:"public_key"`
+	GroupDescription  string `json:"group_description"`
```

Comment:
this should match the below tag. It looks like the api-spec has it as `description` at the moment